packages/styles/scss/components/accordion/_accordion.scss

```diff
@@ -154,11 +153,18 @@ $content-padding: 0 0 0 $spacing-05 !default;
     text-align: start;
   }
 
+  .#{$prefix}--accordion__wrapper {
+    display: grid;
+    // Properties for when the accordion closes
+    padding: 0;
+    grid-template-rows: 0fr;
+    opacity: 0;
+    transition: all $duration-moderate-02 ease-in-out;
```

Comment:
What is the reason for changing this from the animation values it was previously? `$duration-fast-02` and `motion(standard, productive)` instead of `ease-in-out`?

Reply:
I have changed it back to `$duration-fast-02` and `motion(standard, productive)` as suggested